doc/docs/master/how-tos/monitor-job-progress.md

```diff
@@ -0,0 +1,52 @@
+# Monitor Job Progress
+
+After a pipeline starts a job, you can run one of the following commands
+to monitor its status:
+
+* `pachctl list pipeline`
+
+  This command shows all pipelines that run in your cluster
+  and their current status. Jobs for each pipeline will finish with either
+  `success` or `fail` status.
+
+  **Example:**
+
+  ```bash
+  NAME    VERSION INPUT                 CREATED       STATE / LAST JOB    DESCRIPTION
+  montage 1       (edges:/ ⨯ images:/)  2 seconds ago starting / starting A montage pipeline
+  edges   1       images:/*             2 seconds ago running / starting  An edge detection pipeline.
+  ```
+
+* `pachctl list job`
+
+  This command shows jobs that were run for each pipeline. For each job,
+  Pachyderm shows the number of datums in the **PROGRESS** section, the amount
```

Comment:
Might as well re-explain progress here: 

Format of the progress bar is DATUMS PROCESSED + DATUM SKIPPED / TOTAL DATUMS. See (LINK) for more details on datums and when they are skipped versus processed.

Reply:
added